scripts/sendAirdrops.ts

```diff
@@ -0,0 +1,105 @@
+import { ethers } from 'hardhat';
```

Comment:
Rename script to sendBatchTransfer.ts
I would prefer a src/task with clean params instead of abusing ENV